### PR TITLE
mgr/dashboard: include autopep8 for dashboard qa tasks

### PR DESCRIPTION
**Pull request by avanthakkar**
Including autopep8 for the dashboard API tests as well in `tox.ini` 

Signed-off-by: Avan Thakkar <athakkar@redhat.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/tox.ini [hunk 1]
```diff
@@ -147,7 +147,7 @@ deps =
     {[base-lint]deps}
 commands =
     python --version
-    autopep8 {[autopep8]addopts} {posargs:.}
+    autopep8 {[autopep8]addopts} {posargs:. ../../../../qa/tasks/mgr/dashboard}
     isort .
```
Comment (pereman2) [inline src/pybind/mgr/dashboard/tox.ini:151]:
```suggestion
    autopep8 {[autopep8]addopts} {posargs:. ../../../../qa/tasks/mgr/dashboard}
    autopep8 {[autopep8]addopts} .
    isort .
```

Comment (avanthakkar) [reply src/pybind/mgr/dashboard/tox.ini:151]: @pereman2 It covers both paths already

**Comment by avanthakkar** [comment]
jenkins test dashboard cephadm

**Comment by avanthakkar** [comment]
jenkins test dashboard

File: src/pybind/mgr/dashboard/tox.ini (hunk 1 above)
Comment (pereman2) [reply src/pybind/mgr/dashboard/tox.ini:151]: ah true my bad xd

**Review by pereman2 — APPROVED** [review]

**Comment by alfonsomthd** [comment]
jenkins test dashboard cephadm

**Review by alfonsomthd — APPROVED** [review]

**Review by nizamial09 — APPROVED** [review]

**Comment by avanthakkar** [comment]
jenkins test dashboard cephadm

**Review by epuertat — APPROVED** [review]

**Pull request merged**